### PR TITLE
[Issue #3221 & #3222] update footer and identifier to match brand

### LINKED ISSUE #3221: [Brand FE] Footer
### Summary

Update the Footer component to match the [Figma](https://www.figma.com/design/FGxWtAgToKhehLJCiuy1zL/Simpler.Grants.gov-Project?node-id=1521-2210&node-type=frame&t=InZzQfGYsDVKfze4-0).

This first iteration will simply change the colors and spacing to match the brand. Ignore the column layout and content changes in the Figma (which requires content review); it will be addressed in a subsequent Issue. 

Remove the background color from the social icons. 

**Target the [feature/brand](https://github.com/HHS/simpler-grants-gov/tree/feature/brand) branch**

### Acceptance criteria

- [x] Footer includes top double border 
- [x] Footer uses custom background color
- [x] Footer includes new Simpler.Grants.gov logo
- [x] Footer includes more top/bottom padding than the current design
- [x] Social icons have no background color / box

### PR DESCRIPTION
**Pull request by andycochran**
## Summary
Fixes #3221 and #3222

## Changes proposed
Updates the Footer and Identifier components to match the Figma. 

## Context for reviewers
This change resolves both components, which were ticketed separately. 

## Additional information

image

image

### CODE REVIEW

File: frontend/src/components/Footer.tsx [hunk 1]
```diff
@@ -79,42 +79,45 @@ const Footer = () => {
...
-                <a key="email" href={`mailto:${ExternalRoutes.EMAIL_SUPPORT}`}>
-                  {ExternalRoutes.EMAIL_SUPPORT}
-                </a>,
-              ]}
-            />
-          </Grid>
-        </Grid>
+        </GridContainer>
       }
+      secondary={null}
```
Comment (andycochran) [inline frontend/src/components/Footer.tsx:120]: Moved the footer content from secondary to primary. They get different background colors. And they don't collapse when empty. So this avoids random space that'd be in the primary if the content was in secondary. Also, we have no secondary content; all our content is primary.

File: frontend/src/styles/_uswds-theme-custom-styles.scss [hunk 2]
```diff
@@ -279,3 +279,61 @@ i.e.
     }
   }
 }
+
+// Footer
```
Comment (andycochran) [inline frontend/src/styles/_uswds-theme-custom-styles.scss:283]: This took a lot of custom CSS. There aren't a lot of settings options for customizing the Footer and Identifier. Especially wrt layout and color.

File: frontend/src/styles/_uswds-theme-custom-styles.scss [hunk 3]
```diff
@@ -279,3 +279,61 @@ i.e.
...
+}
+
+.usa-footer__social-links,
+.usa-footer__contact-info {
+  justify-content: left;
+
+  @include at-media("tablet") {
+    justify-content: flex-end;
+  }
+}
```
Comment (andycochran) [inline frontend/src/styles/_uswds-theme-custom-styles.scss:334]: On small screens, aligning everything to the left. Each section is done a different way, some w/ text-align, some with flex grid. And neither can be overridden w/ utility classes or settings.

File: frontend/src/styles/_uswds-theme-custom-styles.scss [hunk 4]
```diff
@@ -279,3 +279,61 @@ i.e.
...
+.usa-footer__primary-section {
+  background-color: color("mint-5");
+  border-top: 6px solid color("mint-50");
+  box-shadow: inset 0 6px 0 0 color("mint-20");
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+
+  @include at-media("tablet") {
+    padding-top: calc(1rem + 3vw);
+    padding-bottom: calc(1rem + 3vw);
```
Comment (andycochran) [inline frontend/src/styles/_uswds-theme-custom-styles.scss:293]: A little trick to give more padding the larger the screen.

**Review by mdragon — APPROVED** [review]
👍🏻

**Pull request merged**